src/OpenLoco/src/Graphics/SoftwareDrawingEngine.cpp

```diff
@@ -154,11 +154,16 @@ namespace OpenLoco::Gfx
             return;
         }
 
-        if (scaleFactor > 1.0f)
+        if (scaleFactor != 1.0f)
```

Comment:
With floating point it's always a little bit dangerous to use == or !=. Normally you would use a range. I don't see why we would change this particular conditional

Reply:
It's a pointless change, as I believe it is currently not possible for scaleFactor to be less than 1.0. But if someone wanted to make that happen anyway, my hunch is that there should be interpolation and stuff, so it seemed more right to me :P

What range should be used here?

Reply:
It could be changed to `width == scaledWidth` to do the same thing but without any potential floating point issues 🤔